src/PostgREST/ApiRequest/QueryParams.hs

```diff
@@ -138,9 +140,8 @@ parse isRpcRead qs = do
     onConflict = lookupParam "on_conflict"
     columns = lookupParam "columns"
     order = filter (endingIn ["order"] . fst) nonemptyParams
-    limits = filter (endingIn ["limit"] . fst) nonemptyParams
-    -- Replace .offset ending with .limit to be able to match those params later in a map
-    offsets = first (replaceLast "limit") <$> filter (endingIn ["offset"] . fst) nonemptyParams
+    offset = lookupParam "offset"
+    limit = lookupParam "limit"
     lookupParam :: Text -> Maybe Text
```

Comment:
This should still allow doing something like `/clients?select=*,projects(*)&projects.limit=1`. I think this change breaks that.